unity/EditorPlugin/AfterUnity56/Initialization.cs

```diff
@@ -71,7 +71,7 @@ public class SpawnManagerScriptableObject : ScriptableObject
         public Vector3[] spawnPoints;
     }
                  */
-                var hasDirtyUserAssets = Resources.FindObjectsOfTypeAll<Object>()
+                var hasDirtyUserAssets = Resources.FindObjectsOfTypeAll<ScriptableObject>()
```

Comment:
Does this check only apply to `ScriptableObject`? I guess other objects should belong to the scene. What about prefabs?

Reply:
I made some experiments and it showed that prefabs are auto-saved.